schema.json

```diff
@@ -199,8 +200,9 @@
           "sourcePass": {
              "type": "string",
              "format": "password",
+             "pattern" : "^[^\"\\'\\s]*$",
              "prettyName": "Source Connection Password",
-             "description": "* Required if Delphix is taking backups",
+             "description": "* Required if Delphix is taking backups. Double quote, single quote or space characters are not valid.",
```

Comment:
"*" here in the starting of the description. Is that required?

Reply:
It was already there. I guess just as a visual aid to show that this is a required thing. I think it's pretty functional with the current UI wizard where we have so many fields. I think we should keep it